### PR TITLE
Add README

### PR DESCRIPTION
**Pull request by ArquintL**
This is a first draft for the README with some instructions that I could remember.
@jcp19 do you had to do additional steps when recently configuring Gobra?

### CODE REVIEW

**Comment by Felalolf** [comment]
I wrote instructions in our Gobra folder.

**Comment by ArquintL** [comment]
@Felalolf Thanks for the pointer. I've updated them correspondingly. Instructions for the IDE will be added as soon as it is released

**Review by Felalolf — COMMENTED** [review]
I think for Boogie, you need to do something with mono as well. But @jcp19 should know more abou that.

**Comment by jcp19** [comment]
Yes, you need Mono if you want to use the Carbon backend.

**Comment by ArquintL** [comment]
Thanks! I've added a corresponding notice and also mention that this is only necessary when choosing carbon

File: README.md [hunk 1]
```diff
@@ -39,14 +39,16 @@ We call annotated Go programs Gobra programs and use the file extension `.gobra`
...
         - viperserver -> ..\viperserver
-4. Install Z3 and Boogie
-    - Get a Z3 executable. A precompiled executable can be downloaded [here](https://github.com/Z3Prover/z3/releases). 
+4. Install Z3 and Boogie. 
+    Steps (iii) and (iv) are specific to Boogie and only necessary when using Carbon as verification backend.
+    1. Get a Z3 executable. A precompiled executable can be downloaded [here](https://github.com/Z3Prover/z3/releases). 
       We tested version 4.8.6 64-Bit.
-    - Set an environmental variable `Z3_EXE` to your Z3 executable.
-    - Get a Boogie executable. Instructions for compilation are given [here](https://github.com/boogie-org/boogie). 
+    2. Set an environmental variable `Z3_EXE` to your Z3 executable.
```
Comment (Aurel300) [inline README.md:46]:
```suggestion
    2. Set the environment variable `Z3_EXE` to the path to your Z3 executable.
```

File: README.md [hunk 2]
```diff
@@ -39,14 +39,16 @@ We call annotated Go programs Gobra programs and use the file extension `.gobra`
...
+    3. Get a Boogie executable. Instructions for compilation are given [here](https://github.com/boogie-org/boogie).
+        [Mono](https://www.mono-project.com/download/stable/) is required on Linux and macOS to run Boogie.
         Alternatively, extract a compiled version from the Viper release tools
         ([Windows](http://viper.ethz.ch/downloads/ViperToolsReleaseWin.zip), [Linux](http://viper.ethz.ch/downloads/ViperToolsReleaseLinux.zip), [macOS](http://viper.ethz.ch/downloads/ViperToolsReleaseMac.zip)).
-    - Set an environmental variable `BOOGIE_EXE` to your Boogie executable.
+    4. Set an environmental variable `BOOGIE_EXE` to your Boogie executable.
```
Comment (Aurel300) [inline README.md:51]:
```suggestion
    4. Set the environment variable `BOOGIE_EXE` to the path to your Boogie executable.
```

**Review by jcp19 — DISMISSED** [review]

**Review by Felalolf — APPROVED** [review]

**Pull request merged**